views.go

```diff
@@ -79,11 +92,12 @@ type LogkeeperOptions struct {
 	// URL is the base URL to append to relative paths.
 	URL string
 	// MaxRequestSize is the maximum allowable request size.
-	MaxRequestSize int
+	MaxRequestSize         int
+	TraceCollectorEndpoint string
```

Comment:
I've never felt terribly strong about this, but as a team Evergreen made an effort to document exported (uppercase) identifiers because then they get pulled into the autogenerated docs.
(the conventions are documented [here](https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit#heading=h.3xlxqknivgzp))